src/main/java/com/minecolonies/items/ItemCompost.java

```diff
@@ -22,6 +23,8 @@
     public ItemCompost()
     {
         super("compost");
+
+        super.setCreativeTab(ModCreativeTabs.MINECOLONIES);
         maxStackSize = 64;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 64 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)